README-zh.md

```diff
@@ -53,102 +53,7 @@ Kmesh创新性的提出将流量治理下沉OS，在数据路径上无需经过
-
-  - Yaml安装方式
-  
-  ```sh
-  # get kmesh.yaml：来自代码仓 deploy/yaml/kmesh.yaml
-  [root@ ~]# kubectl apply -f kmesh.yaml
-  [root@ ~]# kubectl apply -f clusterrole.yaml
-  [root@ ~]# kubectl apply -f clusterrolebinding.yaml
-  [root@ ~]# kubectl apply -f serviceaccount.yaml
-  [root@ ~]# kubectl apply -f l7-envoyfilter.yaml
-  ```
-  
-  默认使用Kmesh功能，可通过调整yaml文件中的启动参数进行功能选择
-  
-- 查看kmesh服务启动状态
-
-  ```sh
-  [root@ ~]# kubectl get pods -A | grep kmesh
-  kmesh-system   kmesh-l5z2j                                 1/1     Running   0          117m
-  ```
-
-- 查看kmesh服务运行状态
-
-    ```sh
-    [root@master mod]# kubectl logs -f -n kmesh-system kmesh-l5z2j
-    time="2024-02-19T10:16:52Z" level=info msg="service node sidecar~192.168.11.53~kmesh-system.kmesh-system~kmesh-system.svc.cluster.local connect to discovery address istiod.istio-system.svc:15012" subsys=controller/envoy
-    time="2024-02-19T10:16:52Z" level=info msg="options InitDaemonConfig successful" subsys=manager
-    time="2024-02-19T10:16:53Z" level=info msg="bpf Start successful" subsys=manager
-    time="2024-02-19T10:16:53Z" level=info msg="controller Start successful" subsys=manager
-    time="2024-02-19T10:16:53Z" level=info msg="command StartServer successful" subsys=manager
-    time="2024-02-19T10:16:53Z" level=info msg="start write CNI config\n" subsys="cni installer"
-    time="2024-02-19T10:16:53Z" level=info msg="kmesh cni use chained\n" subsys="cni installer"
-    time="2024-02-19T10:16:54Z" level=info msg="Copied /usr/bin/kmesh-cni to /opt/cni/bin." subsys="cni installer"
-    time="2024-02-19T10:16:54Z" level=info msg="kubeconfig either does not exist or is out of date, writing a new one" subsys="cni installer"
-    time="2024-02-19T10:16:54Z" level=info msg="wrote kubeconfig file /etc/cni/net.d/kmesh-cni-kubeconfig" subsys="cni installer"
-    time="2024-02-19T10:16:54Z" level=info msg="command Start cni successful" subsys=manager
-  ```
-  
-  更多Kmesh编译构建方式，请参考[Kmesh编译构建](docs/kmesh_compile-zh.md)
-
-- Kmesh L7
-
-  - 安装waypoint
-
-    ```
-    [root@ ~]# istioctl x waypoint apply --service-account default
-    [root@ ~]# kubectl get pods 
-    NAME                                      READY   STATUS         RESTARTS        AGE
-    default-istio-waypoint-6d9df77746-njjq5   1/1     Running        0               10s
-    nginx-55b99db5d6-ddpb2                    1/1     Running        0               10d
-    sleep-865b99bb57-qzjcj                    1/1     Running        0               10d
-    ```
-  
-  - 用kmesh自定义的镜像替换waypoint的原生镜像
-
-    ```
-    [root@ ~]# kubectl get gateway
-    NAME      CLASS            ADDRESS         PROGRAMMED   AGE
-    default   istio-waypoint   10.96.143.232   True         5m7s
-    ```
-
-    在`default` gateway的annotations当中添加`sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint-{arch}:v0.3.0`，将`{arch}`转换为所在宿主机的架构，当前可选的取值为`x86`和`arm`。在gateway pod重启之后，kmesh就具备L7能力了！
+快速入门，尝试构建、部署和体验Kmesh的核心功能，具体请参考[快速入门](./docs/kmesh_quickstart-zh.md)；
```

Comment:
```suggestion
请参考[快速入门](./docs/kmesh_quickstart-zh.md)文档；
```

Reply:
Pls check whether needs to be updated after https://github.com/kmesh-net/kmesh/pull/327 merged.